src/modules/Ticket/ticket.service.ts

```diff
@@ -0,0 +1,14 @@
+import { post, get } from 'apis/apiHelper';
+import { createTicketPayloadType } from './type';
+
+export const postCreateTicket = ({
+  payload,
+}: {
+  payload: createTicketPayloadType;
+}) => {
```

Comment:
As its single line return function we can avoid the return keyword & {}
Also we can just accept the payload as argument & not as a object

export const postCreateTicket = (payload: createTicketPayloadType) => post({ path: '/tickets', requestParams: payload });